adr/ADR-37.md

```diff
@@ -27,37 +27,75 @@ performance.
 
 ## Design
 
-JetStream simplification revolves around the concept of the `Consumer`. The
-consumer definition managed by the server is created by tools (such as nats cli)
-or code. A `Consumer` can be created or added as per the client language
-implementation, typically by providing a stream name and some consumer options.
-
-Clients wanting to consume messages from the consumer, simply locate the
-`Consumer`, which then provides functionality for processing, or consuming
-messages. The representation of the `Consumer` is retrieved, and provides
-operations to `fetch` and `consume` messages.
+The JetStream API consists of three main components: JetStreamContext, Streams,
+and Consumers.
+
+### JetStream Context
```

Comment:
This stuff used to live in JetStream Management Context. Seems unnecessary to change that.